OPHD/States/MapViewStateHelper.cpp

```diff
@@ -402,7 +402,7 @@ void transferProductsPool(ProductPool& source, ProductPool& destination)
 
 	auto& src = source.mProducts;
 
-	for (size_t i = 0; i < PRODUCT_COUNT; ++i)
+	for (std::size_t i = 0; i < static_cast<std::size_t>(ProductType::PRODUCT_COUNT); ++i)
```

Comment:
The `std::` prefix on `std::size_t` is good, but should probably be applied in a separate commit.